test/helpers/createDummyExceptions.ts

```diff
@@ -1,29 +1,30 @@
+import { AnnotatedHearingOutcome } from "@moj-bichard7-developers/bichard7-next-core/core/types/AnnotatedHearingOutcome"
 import exponential from "@stdlib/random-base-exponential"
 import sampleMany from "@stdlib/random-sample"
 import { sample as sampleOne } from "lodash"
+import { HO100206, HO100212, HO100234, HO100301, HO100302, HO100304 } from "./exceptions"
 
-const possibleExceptions = [
-  "HO100321",
-  "HO100304",
-  "HO100314",
-  "HO100301",
-  "HO100310",
-  "HO100302",
-  "HO100402",
-  "HO100404",
-  "HO100212",
-  "HO100324",
-  "HO100206",
-  "HO100325",
-  "HO100104",
-  "HO100322",
-  "HO100323",
-  "HO100306",
-  "HO100234",
-  "HO100101",
-  "HO100325",
-  "HO100113"
-]
+const possibleExceptions: Record<string, (aho: AnnotatedHearingOutcome) => AnnotatedHearingOutcome> = {
+  HO100101: (aho) => aho,
+  HO100104: (aho) => aho,
+  HO100113: (aho) => aho,
+  HO100206: (aho) => HO100206(aho),
+  HO100212: (aho) => HO100212(aho),
+  HO100234: (aho) => HO100234(aho),
+  HO100301: (aho) => HO100301(aho),
+  HO100302: (aho) => HO100302(aho),
+  HO100304: (aho) => HO100304(aho),
+  HO100306: (aho) => aho,
+  HO100310: (aho) => aho,
+  HO100314: (aho) => aho,
+  HO100321: (aho) => aho,
+  HO100322: (aho) => aho,
+  HO100323: (aho) => aho,
+  HO100324: (aho) => aho,
+  HO100325: (aho) => aho,
+  HO100402: (aho) => aho,
+  HO100404: (aho) => aho
```

Comment:
Partially completed list of functions for generating exceptions. Should be straightforward to copy one of the existing functions and edit it appropriately.